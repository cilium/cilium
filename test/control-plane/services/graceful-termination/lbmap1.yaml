AffinityMatch: {}
BackendByID:
  "1":
    FEPortName: ""
    ID: 1
    IP: 10.244.0.3
    NodeName: ""
    Port: 8081
    Protocol: TCP
    RestoredFromDatapath: false
    Scope: 0
    State: 0
DummyMaglevTable: {}
ServiceByID:
  "1":
    Backends:
    - FEPortName: ""
      ID: 1
      IP: 10.244.0.3
      NodeName: ""
      Port: 8081
      Protocol: TCP
      RestoredFromDatapath: false
      Scope: 0
      State: 0
    Frontend:
      ID: 1
      IP: 10.96.221.149
      Port: 8081
      Protocol: NONE
      Scope: 0
    HealthCheckNodePort: 0
    L7LBFrontendPorts: null
    L7LBProxyPort: 0
    LoadBalancerSourceRanges: null
    Name: ""
    Namespace: ""
    NatPolicy: ""
    SessionAffinity: false
    SessionAffinityTimeoutSec: 0
    TrafficPolicy: ""
    Type: ClusterIP
SourceRanges: {}
SvcActiveBackendsCount:
  "1": 1
