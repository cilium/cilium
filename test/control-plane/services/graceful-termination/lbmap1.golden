 - Services -------------------------------------------
| ID |      Type |              Frontend | Backend IDs |
|----+-----------+-----------------------+-------------
|  0 | ClusterIP | 10.96.24.51:8081/NONE |           0 |
 ------------------------------------------------------

 - Backends -----------------------------------
| ID |            L3n4Addr |  State | Restored |
|----+---------------------+--------+----------
|  0 | 10.244.0.3:8081/TCP | active |    false |
 ----------------------------------------------

